src/tests/Microsoft.PowerFx.Core.Tests/NamedFormulaTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Syntax;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class NamedFormulaTests
+    {
+        private readonly string script = "x=1;y=2;";
```

Comment:
in your next PR for parser errors, we'll need a lot more tests, converting this to use [Theory] and [InlineData] would be good.

Reply:
Changed